src/EVault/EVault.sol

```diff
@@ -226,7 +226,7 @@ contract EVault is Dispatch {
 
     function setHookConfig(address newHookTarget, uint32 newHookedOps) public virtual override use(MODULE_GOVERNANCE) {}
 
-    function setLTV(address collateral, uint16 ltv, uint32 rampDuration) public virtual override use(MODULE_GOVERNANCE) {}
+    function setLTV(address collateral, uint16 borrowLTV, uint16 targetLiquidationLTV, uint32 rampDuration) public virtual override use(MODULE_GOVERNANCE) {}
```

Comment:
Can we make the parameter name here just `liquidationLTV`? Most people won't know/care about the target or ramping part. I'd probably also rename it in the event, and make "target" internal-only (except for `LTVFull` -- this is an internal dumping method anyway).

Reply:
I refactored a little bit more due to declaration shadowing warnings and for (in my opinion) improved clarity